### DISCUSSION TITLE
I've lost one of my authenticators

### DISCUSSION THREAD

**Discussion by VCalidor** [community/community#123486] [Other Feature Feedback, Questions, & Ideas]
Please help, my account has 2 2FA authentifications, Github mobile and app authenticators, i still have my password and Github mobile but they're not enough.

   I tried to send a message to support but It always says, "Something went wrong", dont know what else to do.

**Comment by CiceroLino** [comment]
Oh, boy. Why do you have 2 app authenticators to same account?

I never seem anything like this but can you have tried change any setting just using your original email? Restore your second authenticator or removing from your account (if are these VCalitor)

**Reply by VCalidor** [reply]
I don't know why i did that either lol, but when i try to remove that second authenticator they request me to authenticate with him

**Reply by CiceroLino** [reply]
Sorry for late reply. But have you tried to use github mobile? You may dont know but he also can do two-factor authentication, and maybe you can remove one of those authenticators.

## Like that

![image](https://github.com/community/community/assets/53065315/8fe20d41-3805-4dca-bae6-244436989d1e)
